apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -251,3 +251,18 @@ function load_block_patterns() {
 	Block_Patterns::get_instance();
 }
 add_action( 'plugins_loaded', __NAMESPACE__ . '\load_block_patterns' );
+
+
+/**
+ * Load Premium Content Block
+ */
+function load_premium_content() {
+	/**
+	 * Disabled until we're ready to disable the premium content plugin in mp-plugins/earn
+	 */
+	if ( function_exists( '\A8C\FSE\Earn\PremiumContent\premium_content_block_init' ) ) {
```

Comment:
Just out of curiosity, if the php file with this function hasn't been required yet, won't the function never exist, and therefore the code will always be loaded?

Reply:
Code is actually running on wpcom already.

Reply:
ah I see what you mean, make sense :) 👍 

By the way, this plugin is also loaded on Atomic sites. I assume there's also an existing function there, or you do want it to load there?

Reply:
> By the way, this plugin is also loaded on Atomic sites. I assume there's also an existing function there, or you do want it to load there?

At this point, yes, we want it to load on Atomic as is. This function check is specifically for what will be short period in which sandboxes will have the same code included via `mu-plugins/earn/premium-content`.

Reply:
To expand a bit more. Shortly after landing this PR, a WordPress sandbox commit will be made to disable including any of the PHP that currently lives at `mu-plugins/earn/premium-content` and is a copy of what will now be in the FSE plugin.

Reply:
Wonderful, thank you for clarifying! That makes a lot of sense.